dokang/utils.py

```diff
@@ -20,7 +26,7 @@ def doc_set(settings, uploaded):
     harvester = get_harvester(settings['dokang.uploaded_docs.harvester'])
     upload_dir = settings.get('dokang.uploaded_docs.dir')
 
-    uploaded = uploaded.decode('utf-8')
+    uploaded = uploaded.decode('utf-8') if isinstance(uploaded, bytes) else uploaded
```

Comment:
If I understand correctly, `uploaded` is a bytestring only when `doc_set()` is called from `load_doc_sets()` where it is the result of `os.listdir()` which returns bytes in Python 2 and a string in Python 3. Perhaps we should decode only where and when it is necessary (i.e. under Python 2 in `load_doc_sets()`)? That would look less random.